requirements/requirements-production.txt

```diff
@@ -1,4 +0,0 @@
--r requirements.txt
-gunicorn==19.5.0
```

Comment:
In main(pip-freeze write all of this by default). I think it's better to have only 2. Anyway we use all of this in any container